src/__tests__/lib/containers/EntityIdList.test.tsx

```diff
@@ -3,36 +3,39 @@ import { mockAllIsIntersecting } from 'react-intersection-observer/test-utils'
 import EntityIdList, {
   EntityIdListProps,
 } from '../../../lib/containers/EntityIdList'
-import { mount } from 'enzyme'
 import { act } from 'react-dom/test-utils'
-import { SynapseTestContext } from '../../../mocks/MockSynapseContext'
+import {
+  MOCK_CONTEXT_VALUE,
+  SynapseTestContext,
+} from '../../../mocks/MockSynapseContext'
+import { render, screen, waitFor } from '@testing-library/react'
+import { mockFileEntityHeader } from '../../../mocks/entity/mockEntity'
+import { getEntityHeadersByIds } from '../../../lib/utils/SynapseClient'
+
+const SynapseClient = require('../../../lib/utils/SynapseClient')
+
+SynapseClient.getEntityHeadersByIds = jest
+  .fn()
+  .mockResolvedValue({ results: [mockFileEntityHeader] })
 
 describe('EntityIdList: basic functionality', () => {
```

Comment:
This test suite was giving me issues so I altered it a bit

Instead of two enzyme tests that 
* verify rendering
* mock useState

I replaced this with one RTL test that mocks SynapseClient and verifies that the state string is visible in the rendered component